keadm/cmd/keadm/app/cmd/util/k8sinstaller.go

```diff
@@ -146,11 +153,7 @@ func installCRDs(ks *K8SInstTool) error {
 			}
 
 			// not found err, create crd from crd file
-			if dir == "router" {
-				err = createKubeEdgeV1CRD(crdClient, KubeEdgeCrdPath+"/"+crdFile)
-			} else {
-				err = createKubeEdgeV1beta1CRD(crdClient, KubeEdgeCrdPath+"/"+crdFile)
-			}
+			err = createKubeEdgeV1CRD(crdClient, KubeEdgeCrdPath+"/"+crdFile)
```

Comment:
Make `keadm` incompatible, `keadm` need to be compatible with lower version, e.g. `1.8.0`

Reply:
If we want to keep compatible with versions before, it's need to make explicit version judgments here, is it valuable? @llhuii

Reply:
> Make `keadm` incompatible, `keadm` need to be compatible with lower version, e.g. `1.8.0`

we can  use keadm1.8.0 to install kubeedge 1.8.0  :) so we don't need keep keadm 1.9.0 compatible with 1.8.0? 😂

Reply:
Better to be compatible with the the recent two or threee minor versions, e.g. keadm 1.9.0 can install 1.8.0/1.7.0

Reply:
I think we need to keep compatible with versions before, so please add the version judgments here, though the code is a bit ugly.

Reply:
We can use this lib to do the comparison work: github.com/blang/semver

Reply:
> we can use keadm1.8.0 to install kubeedge 1.8.0 :) so we don't need keep keadm 1.9.0 compatible with 1.8.0? 😂


Questions for this:
1. What does the `--kubeedge-version` option of `keadm` mean?
2. And `keadm` 1.9.0 has the default `1.8.2` for `kubeedge-version` instead of `1.9.0`

More user-friendly to switch different versions by using the same keadm binary.
So I think `--kubeedge-version` is useful.

Reply:
In other way we can use k8s dynamic client to solve crd v1 and v1beta1 branch code

Reply:
GVR is not easy to generate, So just use version compare to decide which CRD version we should use to create CRDs

Reply:
1. Unmarshal yaml to unstructured.Unstructured
2. gvk = Unstructured.GetObjectKind().GroupVersionKind()

Reply:
Then look more elegant

Reply:
> 1. Unmarshal yaml to unstructured.Unstructured
> 2. gvk = Unstructured.GetObjectKind().GroupVersionKind()

this is GVK, but not GVR :)
```
// GroupVersionKind unambiguously identifies a kind.  It doesn't anonymously include GroupVersion
// to avoid automatic coercion.  It doesn't use a GroupVersion to avoid custom marshalling
type GroupVersionKind struct {
	Group   string
	Version string
	Kind    string
}
```
```
// GroupVersionResource unambiguously identifies a resource.  It doesn't anonymously include GroupVersion
// to avoid automatic coercion.  It doesn't use a GroupVersion to avoid custom marshalling
type GroupVersionResource struct {
	Group    string
	Version  string
	Resource string
}
```

Reply:
gvk to resource: k8s/io/kubernetes/pkg/api/meta.UnsafeGuessKindToResource